docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,392 @@
+
+#. Run the following command to create an Aiven for PostgreSQL service named ``demo-postgresql``:
+
+   ::
+
+      avn service create demo-postgresql          \
+          --service-type pg                       \
+          --cloud CLOUD_AND_REGION                \
+          --plan business-4                       \
+          --project PROJECT_NAME
+
+#. Run the following command to create an Aiven for Apache Flink service named ``demo-flink``:
+
+   ::
+
+      avn service create demo-flink               \
+          --service-type flink                    \
+          --cloud CLOUD_AND_REGION                \
+          --plan business-4                       \
+          --project PROJECT_NAME
+
+#. Add the ``demo-kafka`` and ``demo-postgresql`` integrations to the ``demo-flink`` service.
+
+   a. Enter the following command to add the ``demo-kafka`` service integration:
+
+      ::
+
+         avn service integration-create           \
+             --project PROJECT_NAME               \
+             --service-type flink                 \
+             -s demo-kafka                        \
+             -d demo-flink
+
+   b. Enter the following command to add the ``demo-postgresql`` service integration:
+
+      ::
+
+         avn service integration-create           \
+             --project PROJECT_NAME               \
+             --service-type flink                 \
+             -s demo-postgresql                   \
+             -d demo-flink
+
+   c. Enter the following command to list the integrations:
+
+      ::
+
+         avn service integration-list demo-flink
+
+      The output should show you that both ``demo-kafka`` and ``demo-postgresql`` integrations are enabled as well as the corresponding ``integration_id`` values that you need later when creating data tables.
+
+
+
+Set up sample data
+------------------
+
+Before you start, clone the `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_ Git repository to your computer and then run ``pip install -r requirements.txt`` in your local copy to make sure that the necessary dependencies are installed.
+
+1. Using the Aiven CLI, run the following command to download the certificate files for your ``demo-kafka`` service:
+
+   ::
+
+      avn service user-creds-download demo-kafka  \
+          --username avnadmin                     \
+          -d TARGET_DIR
+
+   Replace ``TARGET_DIR`` with the path to a local folder on your computer.
+
+#. Run the following command to see the service parameters, then copy the ``host`` and ``port`` values.
+
+   ::
+
+      avn service get kafka-demo --format '{service_uri_params}'
+
+#. Run the following Python command to create the sample records using the `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_ tool:
+
+   ::
+
+      python3 python-fake-data-producer-for-apache-kafka/metricproducer.py \
```

Comment:
Can we replace this with the docker version https://github.com/aiven/fake-data-producer-for-apache-kafka-docker